codespell_lib/_codespell.py

```diff
@@ -965,7 +945,7 @@ def parse_file(
                             "WRONGWORD": cwrongword,
                             "RIGHTWORD": crightword,
                             "REASON": creason,
-                        }
+                        },
```

Comment:
Why not an f-string here toot? I think `{line.strip()}` is OK.

Why add a comma here?

Reply:
Let's forget about the comma, perhaps change to an f-string though?